src/elements/common/sub-header/Sort.js

```diff
@@ -21,7 +22,9 @@ type Props = {
     sortDirection: SortDirection,
 };
 
-const SORT_ITEMS: Array<Array<SortBy>> = [
+type SortItem = [SortBy, SortDirection];
```

Comment:
This was incorrect before, but was working because of the `any` type that it was getting set to.